readthedocs/doc_builder/python_environments.py

```diff
@@ -20,6 +20,11 @@
 
 log = logging.getLogger(__name__)
 
+try:
+    unicode        # Python 2
+except NameError:
+    unicode = str  # Python 3
```

Comment:
We already have a pattern for this using `future`. I believe this should just be:

```
from builtins import str
```

Correct?